.kitchen.docker.yml

```diff
@@ -90,13 +90,13 @@ platforms:
     intermediate_instructions:
       - RUN zypper --non-interactive install aaa_base perl-Getopt-Long-Descriptive which net-tools
 
-suites:
```

Comment:
So we've actually been trying to move back to the .dokken filenames. Sean has been pushing hard to appropriately name the files and he's right here. The only reason we have things that are .docker. was me being lazy and not renaming files.